contrib/local-environment/oauth2-proxy.cfg

```diff
@@ -1,11 +1,16 @@
 http_address="0.0.0.0:4180"
 cookie_secret="OQINaROshtE9TcZkNAm-5Zs2Pv3xaWytBmc5W7sPX7w="
-provider="oidc"
 email_domains="example.com"
-oidc_issuer_url="http://dex.localhost:4190/dex"
+cookie_secure="false"
+upstreams="http://httpbin.localtest.me:8080"
+cookie_domains=[".localtest.me"] # Required so cookie can be read on all subdomains.
+whitelist_domains=[".localtest.me"] # Required to allow redirection back to original requested target.
+
+# dex provider
 client_secret="b2F1dGgyLXByb3h5LWNsaWVudC1zZWNyZXQK"
 client_id="oauth2-proxy"
-cookie_secure="false"
+redirect_url="http://oauth2-proxy.localtest.me:4180/oauth2/callback"
 
-redirect_url="http://localhost:4180/oauth2/callback"
-upstreams="http://httpbin"
+oidc_issuer_url="http://dex.localtest.me:4190/dex"
+provider="oidc"
+provider_display_name="Dex"
```

Comment:
Nit, should have new line as final character in every file

Reply:
resolved